lib/minesweeper_2pl/cli.rb

```diff
@@ -1,4 +1,178 @@
-module Minesweeper_2pl
+module Minesweeper
   class CLI
+    def print(msg)
+      puts msg
+    end
+
+    def start
+      welcome
+      get_player_params
+    end
+
+    def get_move(game)
+      ask_for_move
+      get_player_input(game)
+    end
+
+    def get_player_params
+      result = []
```

Comment:
Is there a Ruby data structure you could use here that would let you give a descriptive name to each component of these params instead of using array indices?